onadata/apps/api/permissions.py

```diff
@@ -163,6 +164,10 @@ class SubmissionReviewPermissions(XFormPermissions):
         'DELETE': ['logger.delete_xform'],
     }
 
+    def _check_is_admin_or_manager(self, user: User, xform: XForm):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/Ona/onadata/pullRequest?prid=5975563)